### PR TITLE
Merge input overhaul into master

### PR DESCRIPTION
**Pull request by chaukap**
Input overhaul is ready to be reviewed and then merged into master.

### CODE REVIEW

File: StepWat_R_Wrapper_Parallel/STEPWAT.Wrapper.MAIN_V3.R [hunk 1]
```diff
@@ -47,13 +46,374 @@ tick_on<-proc.time()
...
+#Append rgroup_template.in withing STEPWAT_DIST to all the created files and save with a unique filename
+for (i in rgroup_files)
+{
+  system(paste("cat ","rgroup_template.in >>",i,sep=""))
+}
+
+setwd("..")
+
+########################### Garbage collection for species, rgroup, and soil requirements #######################
+# All of these variables were created in the species, soils, and rgroup sections. They will never be used again.
```
Comment (chaukap) [inline StepWat_R_Wrapper_Parallel/STEPWAT.Wrapper.MAIN_V3.R]: I will add better documentation to this section to explain why we removed these variables, as well as add a comment letting future programmers know that they should delete their variables as soon as they are no longer needed.

File: inputs/syntax_inputs.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#This file describes the functionality of inputs within rSFSTEP2 and the options that exist for specifying inputs
+
+Three input files are held in the inputs folder:
+1) InputData_Rgroup.csv --> contains rgroup parameters, along with parameters for fire and grazing (top table of rgroup.in only)
+
+2) InputData_SoilLayers.csv --> contains soil depth and soil properties 
+
+3) InputData_Species.ccv --> contains species parameters (top table of species.in only)
```
Comment (chaukap) [inline inputs/syntax_inputs.txt]: Typo here: .ccv -> .csv

File: inputs/syntax_inputs.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The three options:
+1) For fixed inputs for all sites: set "all" in the Site column and a unique identifier to label that set of parameters in the treatment column. The default is fixedn, where n is a number that uniquely identifies that set of fixed inputs (in this cased n=1, so fixed1). However, any unique identifier can be specified. That unique identifier will be used to label output files and added as a column in the output SQLite tables.
+
+2) For fixed inputs for a subset of sites: set the Site column to x,y where x and y correspond to the site IDS of the subset of sites. The default is 1,2, which means that those inputs will be utilized for sites 1 and 2 only. However, this functionality can be used across more than 2 sites. Like 1), the user needs to also set a unique identifier in the treatment column - fixed2 is the default, but any unique identifier can be utilized.
```
Comment (chaukap) [inline inputs/syntax_inputs.txt]: @kpalmqui maybe we should change x,y to x, y, ..., z to make it more clear to the user that they can specify an arbitrary number of sites.

**Comment by chaukap** [comment]
Does the .Rhistory file need to be kept in the repository? From my understanding .RHistory stores all of the commands that have been entered for the current session. Since .RHistory should be unique to every user shouldn't we remove it entirely?

@kpalmqui let me know what you think I don't have a ton of experience in R and I could be wrong about this.

**Comment by chaukap** [comment]
@kpalmqui all of the points you raised should have been resolved except for number 2. Once this has been corrected we should be able to merge into master.

**Comment by kpalmqui** [comment]
> Does the .Rhistory file need to be kept in the repository? From my understanding .RHistory stores all of the commands that have been entered for the current session. Since .RHistory should be unique to every user shouldn't we remove it entirely?
>
@chaukap Yes, it can be removed entirely. It was accidentially committed by a former programmer who was working on this branch.

**Pull request merged**